[docs][quant] Add fx graph mode quant api doc

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55306 [docs][quant] Add fx graph mode quant api doc**

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27567187](https://our.internmc.facebook.com/intern/diff/D27567187)

jerryzh168 committed:
[docs][quant] Add fx graph mode quant api doc

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

vkuzo commented on docs/source/quantization-support.rst:
```diff
@@ -177,9 +177,10 @@ Layers for the quantization-aware training
-* Functions for graph mode quantization:
-  * :func:`~torch.quantization.quantize_jit` - Function for graph mode post training static quantization
-  * :func:`~torch.quantization.quantize_dynamic_jit` - Function for graph mode post training dynamic quantization
+* Functions for fx graph mode quantization:
+  * :func:`~torch.quantization.quantize_fx.prepare_fx` - Function for preparing the model for post training quantization with fx graph mode quantization
+  * :func:`~torch.quantization.quantize_fx.prepare_qat_fx` - Function for preparing the model for quantization aware training with fx graph mode quantization
+  * :func:`~torch.quantization.quantize_fx.convert_fx` - Function for converting a prepared model to a quantized model with fx graph mode quantization
```
nit: should FX be capitalized?

jerryzh168 replied:
sure

jerryzh168 committed:
Update on "[docs][quant] Add fx graph mode quant api doc"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D27567187](https://our.internmc.facebook.com/intern/diff/D27567187)

[ghstack-poisoned]
